radar/languages-and-frameworks/spring-boot.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Spring Boot"
+ring: adopt
+quadrant: "languages-and-frameworks"
+tags: [java, backend]
+---
+
+Spring Boot is an open source Java-based framework used to create a microservices. It is developed by Pivotal Team and is used to build stand-alone and production ready spring applications.
+
+The Spring Cloud ecosystem also gives you a lot of extension points for developing, deploying and running cloud applications.
+
+It's based on the rock-solid Spring framework and provides excellent documentation.
+
+[Spring Boot](https://spring.io/projects/spring-boot)
+
+### Key Features
+
+- **Great community support**: the large community behind spring boot and Pivotal itself allow for ready-to-use integrations for most commonly used technologies.
+
+- **Production ready**: provide production-ready features such as metrics, health checks, and externalized configuration
+
+- **Fast bootstrapping**: a lot of 'starter' dependencies to simplify your build configuration and make service ready to run
+
+### Our use cases
+- [File Reporter](https://github.com/pagopa/rtd-ms-file-reporter): microservice responsible to provide an aggregated view of a file processing system. Thanks to Spring Boot and it's 'starter' dependencies the microservice expose a REST interface and integrate a kafka consumer in a simple and configurable way.
```

Comment:
@manuraf @GiovanniMancini could you add some other good example of Spring boot usage?

Reply:
In PDV we are using Spring boot framework for three different micro-services:
- https://github.com/pagopa/pdv-ms-tokenizer
- https://github.com/pagopa/pdv-ms-person
- https://github.com/pagopa/pdv-ms-user-registry

Reply:
you could add there repositories, for now :
* https://github.com/pagopa/selfcare-ms-core
* https://github.com/pagopa/selfcare-ms-party-registry-proxy
* https://github.com/pagopa/selfcare-dashboard-backend

Can you edit this branch?

Reply:
we can add this link that lists all repos: https://github.com/search?q=org%3Apagopa+path%3A**%2Fpom.xml+spring-boot+&type=code

Reply:
Would you prefer to include a list of microservices or a link to a github search page with filters set?

Reply:
@iwoak what about this?

Reply:
I've added a reference to github search page with filters set. If it's ok we can resolve this conversation